jwst_mast_query/jwst_query.cfg

```diff
@@ -0,0 +1,78 @@
+# specify the instrument. The default set in the code is nircam
+instrument: nircam
+
+# define the output filename
+outrootdir: $JWSTDOWNLOAD_OUTDIR
+outsubdir: 
+skip_propID2outsubdir: False
+skip_check_if_outfile_exists: False
+
+# specify the list of filetypes to select in the product table, 
+# e.g., _uncal.fits or _uncal.jpg.  If only letters, then _ and .fits are added, 
+# for example uncal gets expanded to _uncal.fits. Typical image filetypes are uncal, rate, rateints, cal
+# Note: with -vv, a list of all filetypes of the observations is shown on screen after the ProductTable
+# is queried for the first time. Example: 
+# List of all filetypes of obtained products: ['_segm.fits', '_asn.json', '_pool.csv', '_i2d.jpg', '_thumb.jpg', '_cat.ecsv', '_i2d.fits', '_stream.fits', '_uncal.fits', '_uncal.jpg', '_cal.fits', '_trapsfilled.fits', '_cal.jpg', '_rate.jpg', '_rateints.jpg', '_trapsfilled.jpg', '_rate.fits', '_rateints.fits']
+filetypes: ['uncal']
+
+# if guidestars is set to True, guidestar products are also included
+guidestars: False
+
+# lookback time in days 
+# Note that all other time parameters (mjdmin,mjdmax,mjd_limits,lre?) override the lookback time.
```

Comment:
Would be good to explain here what the lookback time is (i.e. that it is the number of days prior to today that are searched).

Reply:
Handled in documentation (confluence) for now.